src/services/leafletMarkersHelpers.js

```diff
@@ -101,8 +106,8 @@ angular.module('ui-leaflet').service('leafletMarkersHelpers', function ($rootSco
     };
 
     var _resetMarkerGroup = function (groupName) {
-        if (isDefined(groups[groupName])) {
-            groups.splice(groupName, 1);
+        if (angular.isDefined(groups[groupName])) {
```

Comment:
Yes, it does. But it seems to be unclear whether isDefined was defined in this scope or is a global function. Using angular.isDefined is obvious that we use isDefined from global object angular.